_config.yml

```diff
@@ -605,8 +605,11 @@ rating:
 
 # Show number of visitors to each article.
 # You can visit https://leancloud.cn get AppID and AppKey.
+# Set up instruction: https://leaferx.online/2018/02/11/lc-security/
+# Dependencies: https://github.com/theme-next/hexo-leancloud-counter-security
 leancloud_visitors:
   enable: false
+  security: false
```

Comment:
Why not default to `true`?

Reply:
Because need to install plugin separately.

Reply:
Yes, I mean just like the gitm**i**nt, default to true, and ask the user to install the plugin.

Security is important, isn't it?

Reply:
Important, but LC can work without security plugin.